src/app.js

```diff
@@ -3,18 +3,23 @@
 const _ = require('lodash');
 const prompt = require('prompt');
 const colors = require('colors');
-const kcomb = require('foreach-combination');
 
 // Get essential data
 const monsters = require('./lib/monsters.js');
 const bosses = require('./lib/bosses.js');
 const specials = require('./lib/specials.js');
 const friends = require('./lib/friends.js');
 const debug = require('./debug.js');
+const mapEngine = require('./map-engine.js');
 const config = require('./config.js');
 const battleEngine = require('./battle-engine.js');
 const player = require('./lib/player.js');
 
+// Map properties
```

Comment:
Like I mentioned on `map-engine.js`, you will have to update the `var map` here to import `map` from `map-engine.js` and remove the variables `map_width` and `map_height`.
```suggestion
// Map properties
var map = mapEngine.map;


```